src/main/webapp/jsp/instructorFeedbackResultsTop.jsp

```diff
@@ -244,6 +244,11 @@
     <input type="hidden" name="<%=Const.ParamsNames.USER_ID%>" 
         value="<%=data.account.googleId%>">
 </form>
+    <%if(data.selectedSection.equals("All") && showAll==false){%>
+        <div id="statusMessage" class="alert alert-warning" style="margin-top:20px;" >
+            This session seem to have a large number of responses. It is recommended to view the results one section at at time.
```

Comment:
We should not have "magic strings" like this in the jsp. Try creating a constant in Const.java?